### DISCUSSION TITLE
Only the RTSP

### DISCUSSION THREAD

**Discussion by tv4you2016** [roleoroleo/yi-hack-Allwinner-v2#810] [General]
Hello, is there any possibility of putting only the RTSP service to work?
I don't need any type of recording or page!

**Comment by roleoroleo** [comment]
You can disable all not necessary services from the web gui.
I recommend that you leave at least one way to connect to the camera: ssh or www.

**Reply by tv4you2016** [reply]
Ok 
How can I change the video codec from
H264 to H265??

**Reply by roleoroleo** [reply]
There is an experimental method that reconfigures the cam in h264/h265, editing mtdblock7 partition.
https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/484

Warning:
- It's not tested for all models.
- I don't know if h265, after the switch, is fully supported by grabber/rtsp_server.

**Reply by tv4you2016** [reply]
I realized that there is the possibility of:
Save video when motion is detected:

-If enabled, video will be saved only when motion is detected.

-If disabled, the video will always be saved.

Is there no way to disable both??
basically what I wanted was just RTSP without any recordingI realized that there is the possibility of:
Save video when motion is detected:

-If enabled, video will be saved only when motion is detected.

-If disabled, the video will always be saved.

Is there no way to disable both??
basically what I wanted was just RTSP without any recording


![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/83761813/1e1151b7-2526-448a-a100-b32cfe35acfa)

**Reply by DrioQio** [reply]
Take a look at Configurations > General > Disable Cloud and Recording without Cloud
